src/Escape.elm

```diff
@@ -1,32 +1,28 @@
-module Escape exposing (sanitizeNamespace, sanitize)
+module Escape exposing (sanitize, sanitizeNamespace)
 
 import Regex exposing (Regex)
 
 
-sanitizeNamespace : String -> String
-sanitizeNamespace str =
-    let
-        sanitizeRegex =
-            Regex.regex "[^a-z0-9\\-_]"
-                |> Regex.caseInsensitive
+caseInsensitiveOption : Regex.Options
+caseInsensitiveOption =
+    { caseInsensitive = True
+    , multiline = False
```

Comment:
Well, because it was originally only ```...|> Regex.caseInsensitive``` so I mimic only the `caseInsesitive` behaviour. Should I turn the multiline option to `True` as well?

Reply:
hmm. does it meant that if I use a multiline string as arg it breaks?

Reply:
Probably yes, not sure about that. However those regexes are defined by us and there is no input or possibility to change them from the outside, so I believe it is safe. And it was set the same in previous version.

Reply:
ok... lets wait for a bit